sources/packages/api/src/services/education-program-offering/education-program-offering.service.models.ts

```diff
@@ -0,0 +1,17 @@
+import * as dayjs from "dayjs";
+export class EducationProgramOfferingModel {
+  id: number;
+  name: string;
+  studyStartDate: Date;
+  studyEndDate: Date;
+  offeringDelivered: string;
+  get studyDates(): string {
+    if (this.studyStartDate === null) {
+      return "No program dates";
+    } else {
+      return `${dayjs(this.studyStartDate).format("MMMM, D YYYY")} - ${dayjs(
```

Comment:
Maybe would be good to create a constants.ts file inside the utilities to have the formats in the same place.

/utilitites/constants.ts
```
// Expected output: January, 9 2021
export const EXTENDED_DATE_FORMAT= "MMMM, D YYYY";
```

Reply:
That would be cool, Done